UnityProject/Assets/Scripts/Systems/GhostRoles/GhostRoleManager.cs

```diff
@@ -143,7 +143,7 @@ public void ServerUpdateRole(uint key, int minPlayers, int maxPlayers, float tim
 			role.UpdateRole(minPlayers, maxPlayers, timeRemaining, playerCount);
 
 			// Will be exactly -1 if timeout is set as indefinite.
-			if (timeRemaining <= 0 && timeRemaining != -1)
```

Comment:
Must've missed where `timeRemaining` is `-2` - what does it do?

Reply:
![image](https://user-images.githubusercontent.com/56727168/181270450-168f2681-a5d2-409b-ad0e-e3cf8e45515e.png)

Reply:
Oh wait i'm an idiot